test/validateProject.ts

```diff
@@ -142,6 +142,24 @@ export function getCSharpScriptValidateOptions(runtime: ProjectRuntime = Project
     };
 }
 
+export function getPowerShellValidateOptions(): IValidateProjectOptions {
+    return {
+        expectedSettings: {
+            projectLanguage: ProjectLanguage.PowerShell,
+            projectRuntime: ProjectRuntime.v2,
+            deploySubpath: '.',
+            preDeployTask: 'func: extensions install',
+            templateFilter: 'All'
+        },
+        expectedPaths: [
+            'profile.ps1'
```

Comment:
`profile.ps1` is created when doing:

```
func init --worker-runtime powershell
```

how can I reflect that in the test?

Reply:
If `func init` writes that file, we should probably also write it in `PowerShellProjectCreator.onCreateNewProject`. You can take a look [here]( https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/commands/createNewProject/ScriptProjectCreatorBase.ts#L170) for an example. (And it should be included in the test)

We don't run `func init` directly for a variety of reasons, but that does result in a bit of duplicated logic (like copying profile.ps1 to this repo). The `func` cli is only really required for debugging and we have a notable portion of users that create/deploy without ever debugging haha. Plus it's just slower to use and harder to parse errors.

Reply:
A discussion for another time... but stuff like this can be pulled from https://github.com/Azure/azure-functions-core-tools/tree/dev/src/Azure.Functions.Cli/StaticResources

Maybe worth doing that?

Reply:
What do you mean by pulling it from there? I suggest just copying it to this repo. IMO the project-level files don't change enough to be worth any sort of dynamic pulling. Let me know if you think `profile.ps1` would be different

Reply:
I think that's fair :)